nielvis/academicIO.py

```diff
@@ -225,10 +245,128 @@ def read(self, *args):
         if args_len == 0:
             return self.__read_single_point()
         elif args_len == 2:
-            return self.__read_multiple_points(args[0], args[1])
+            if self.is_continuous['A'] or self.is_continuous['B']:
+                return self.__read_multiple_points_continuous(args[0], args[1])
+            else:
+                return self.__read_multiple_points_n_samples(args[0], args[1])
         else:
             raise TypeError('read() takes either 0 (single point) or 2 (multiple points) arguments, but given %d' % args_len)
 
+    def start_continuous_mode(self, sample_rate):
+        """
+        Configure the sample rate and start the acquisition. 
+
+        Args:
+            sample_rate (number):
+                Specifies the sampling frequency, in hertz, of the input
+                signal. 
+                If you select only one channel, the valid range for sample
+                rate is between 1 Hz and 1 MHz. 
+                If you select multiple channels, the valid range for sample
+                rate is between 1 Hz and 250 KHz. 
+        """
+        number_of_channels = len(self.channel_list)
+
+        if number_of_channels == 1:
+            assert 1 <= sample_rate <= 1000000, 'If you select only 1 channel, then the valid range for sample rate is between 1 Hz and 1 MHz.'
+        else:
+            assert 1 <= sample_rate <= 250000, 'If you select multiple channels, then tthe valid range for sample rate is between 1 Hz and 250 KHz.'
+
+        self.__sample_rate = sample_rate
+
+        def __open_ai_continuous(configuration):
+            for bank in Bank:
+                bank = bank.value
+                if configuration[bank]['numberOfChannels'] > 0:
+                    assert not AnalogInput.is_continuous_started[bank], 'Cannot open the reference to channels on one bank using more than one Open function at the same time to perform continuous mode signal acquisition.'
```

Comment:
```suggestion
                    assert not AnalogInput.is_continuous_started[bank], 'Cannot open the reference to channels on one bank using more than one open function at the same time to perform continuous signal acquisition.'
```

I am not sure whether we should capitalize "O" in "Open". You use "read function", not "Read function" elsewhere.